content/docs/02-getting-started/01-navigating-the-library.mdx

```diff
@@ -0,0 +1,66 @@
+
+RSCs and Server Actions are still in early development. When considering whether to use AI SDK RSC, it's important to be aware of the current limitations of RSCs and Server Actions:
+
+- **Cancellation/Logging/Debugging**: currently, it is not possible to abort a stream using Server Actions. This will be improved in future releases of React and Next.js.
+- **Quadratic Data Transfer**: using [`createStreamableUI`](/docs/reference/ai-sdk-rsc/create-streamable-ui) can lead to quadratic data transfer. You can avoid this using [ `createStreamableValue` ](/docs/reference/ai-sdk-rsc/create-streamable-value) instead, and rendering the component client-side.
+- **Re-mounting Issue During Streaming**: when using `createStreamableUI`, there is a noticeable flash that occurs when components re-mount on `.done()`, causing undesirable behavior.
+- **Human-in-the-Loop Roundtrips**: currently there is no support for human-in-the-loop chatbot roundtrips with AI SDK RSC.
+
+If any of the above limitations are important to you application, we recommended using AI SDK UI.
+
+### AI SDK UI Framework Compatibility
+
+AI SDK UI supports the following frameworks: [React](https://react.dev/), [Svelte](https://svelte.dev/), [Vue.js](https://vuejs.org/), and [SolidJS](https://www.solidjs.com/). Here is a comparison of the supported functions across these frameworks:
+
+| Function                                                   | React               | Svelte              | Vue.js              | SolidJS             |
+| ---------------------------------------------------------- | ------------------- | ------------------- | ------------------- | ------------------- |
+| [useChat](/docs/reference/ai-sdk-ui/use-chat)              | <Check size={18} /> | <Check size={18} /> | <Check size={18} /> | <Check size={18} /> |
+| [useChat](/docs/reference/ai-sdk-ui/use-chat) tool calling | <Check size={18} /> | <Cross size={18} /> | <Cross size={18} /> | <Check size={18} /> |
+| [useChat](/docs/reference/ai-sdk-ui/use-chat) attachments  | <Check size={18} /> | <Cross size={18} /> | <Cross size={18} /> | <Cross size={18} /> |
+| [useCompletion](/docs/reference/ai-sdk-ui/use-completion)  | <Check size={18} /> | <Check size={18} /> | <Check size={18} /> | <Check size={18} /> |
+| [useObject](/docs/reference/ai-sdk-ui/use-object)          | <Check size={18} /> | <Cross size={18} /> | <Cross size={18} /> | <Cross size={18} /> |
+| [useAssistant](/docs/reference/ai-sdk-ui/use-assistant)    | <Check size={18} /> | <Check size={18} /> | <Check size={18} /> | <Cross size={18} /> |
```

Comment:
Can you add the info box that we have on the other 2 compatibility charts as well (to encourage people to contribute)?